### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
- Fixes typos in `tests/`.
- Improves the `typos` configuration.

### CODE REVIEW

File: tests/class-php-typography-test.php [hunk 1]
```diff
@@ -933,8 +933,8 @@ public function test_smart_quotes_off( $html, $result ) {
 	 * @uses PHP_Typography\Text_Parser\Token
 	 */
 	public function test_smart_quotes_french_should_not_apply() {
-		$html   = 'attributs <code>role="group"</code> et <code>aria-labelledby</code>';
-		$result = 'attributs <code>role="group"</code> et <code>aria-labelledby</code>';
+		$html   = 'attributes <code>role="group"</code> et <code>aria-labelledby</code>';
+		$result = 'attributes <code>role="group"</code> et <code>aria-labelledby</code>';
```
Comment (mundschenk-at) [inline tests/class-php-typography-test.php:938]: This is an incorrect change as the sentence is in French.

**Comment by mundschenk-at** [comment]
Thank you! Apart from the incorrect line in `class-php-typography-test.php` this looks good. Could you please rebase it on the latest `main` and flatten the commits (one for the typos in the test cases – prefixed with `Tests:` – and one for the config change – prefixed with `Meta:`)?

File: tests/class-php-typography-test.php (hunk 1 above)
Comment (szepeviktor) [reply tests/class-php-typography-test.php:938]: Oh no! I am didn't see "et" in that line.

Comment (szepeviktor) [reply tests/class-php-typography-test.php]: Done 🍏

**Comment by szepeviktor** [comment]
> Could you please rebase it on the latest `main` and flatten the commits

It is already on `main`.
I cannot flatten this PR as `git rebase` goes crazy because of the merge commit.

**Comment by szepeviktor** [comment]
A PR branch is writable by **you**.

Okay I do it but I have to use a computer 🙃

**Comment by mundschenk-at** [comment]
Mhm, I'll have a look if I can rewrite history here (`rebase -i`).

File: tests/class-php-typography-test.php (hunk 1 above)
Comment (mundschenk-at) [reply tests/class-php-typography-test.php:938]: Mhm, somehow I still see the old version here.

**Comment by szepeviktor** [comment]
I have github.com connectivity issues ...

**Comment by szepeviktor** [comment]
I'm deep down in the rabbit hole 🩸🐇

**Comment by szepeviktor** [comment]
No one expected a Git Course!!

**Comment by szepeviktor** [comment]
Done 🍏 

_I need to eat!_

**Comment by mundschenk-at** [comment]
Thank you!

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**